leaper.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, *args, **kwargs):
     def prepare_review(self):
         # SLE workflow is not concerned with origin.
```

Comment:
Remove this comment since that does not seem to be the case given these changes.